go-controller/pkg/cni/utils.go

```diff
@@ -117,13 +117,18 @@ func PodAnnotation2PodInfo(podAnnotation map[string]string, podNADAnnotation *ut
 		return nil, err
 	}
 
+	// Skip default network IP config if pod is annotated with it
+	_, hasSkipIPConfigAnnotation := podAnnotation[SkipIPConfigAnnotation]
```

Comment:
> I though we were not going to have this annotation in the end.

Have leave like the enhancement, but if @maiqueb is ok removing the flag (since multi homing will use NAD) I can just check if pod is live migratable and remove this.

> When it is being set?

When the VirtualMahchine is created for example by the hypershift operator.